lib/manipulateFile.rb

```diff
@@ -0,0 +1,21 @@
+class ManipulateFile
+    
+    attr_reader :file
+
+    def initialize(file)
+        begin
+            @file = File.readlines(file, chomp: true)
+        rescue Errno::ENOENT
+            puts "File doesn't exist."
+        end
+    end
+
+    def getFirstLine
```

Comment:
Por convenção no ruby nós declaramos funções seguindo o formato `snake_case`
```suggestion
    def get_first_line
```